bin/yaml/c.yaml

```diff
@@ -71,6 +71,20 @@ compilers:
         - 4.3.0
         - 4.4.0
         - 4.5.0
+    z80-cclang:
```

Comment:
This part is not needed, it is fully covered by the cpp install part

Reply:
Alright, I'll remove it.